USFMToolsSharp/Models/Markers/QMarker.cs

```diff
@@ -14,8 +14,13 @@ public class QMarker : Marker
         public override string Identifier => "q";
         public override List<Type> AllowedContents => new List<Type>() {
             typeof(BMarker),
-            typeof(QSMarker),

```

Comment:
Fairly certain qs is still valid in a q marker

Reply:
Yeah you're right! I just looked at the specs again.